hms-app/src/components/HackathonHeader.tsx

```diff
@@ -32,6 +32,7 @@ export default function HackathonHeader(props: IProps) {
               readOnly
               value={hackathonData.description || ''}
               id='hackathonDescriptionEditor'
+              style={{ color: 'gray', backgroundColor: 'transparent', border: 'none', textAlign: 'center' }}
```

Comment:
I think we should not align this to 'center' as we might have lists in the description and this could look weird